### PR TITLE
Bump dcos-mesos to latest master a4b1134.

### PR DESCRIPTION
**Pull request by adam-mesos**
## High Level Description

Bump Mesos to latest master, to pick up storage-related (CSI/SLRP) changes, and others.

## Related Issues

  - [DCOS_OSS-1970](https://jira.mesosphere.com/browse/DCOS_OSS-1970) Bump mesos to pre-1.5.0 in preparation for beta3
  - [MESOS-7235](https://issues.apache.org/jira/browse/MESOS-7235) Resource Provider and CSI
  - [DCOS-13242](https://jira.mesosphere.com/browse/DCOS-13242) Resource Provider (RP) abstraction in Mesos
  - [DCOS-13241](https://jira.mesosphere.com/browse/DCOS-13241) Container Storage Interface

## Checklist for all PR's

  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [X] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [X] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

  - [X] Changelog for [dcos-mesos](https://github.com/mesosphere/mesos/compare/96fe7d9...a4b1134)
  - [ ] Test Results: [link to CI job test results for component(s)]

### CODE REVIEW

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by kaysoky** [comment]
Note for whoever can edit the description on this PR...
Here's a changelog link for the Mesos modules bump: https://github.com/dcos/dcos-mesos-modules/compare/9adfabc5d07048f55c3da5a1bad17d2109750f0e...26672593e6e0910c4983cd7d6c8b03f62dd55e2c

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -367,22 +367,65 @@ package:
...
+            "name": "default",
+            "containers": [
+              {
+                "services": [
+                  "CONTROLLER_SERVICE",
+                  "NODE_SERVICE"
+                ],
+                "command": {
+                  "shell": false,
+                  "value": "/opt/mesosphere/packages/active/bin/devices-plugin",
```
Comment (jieyu) [inline gen/dcos-config.yaml]: This should be
`/opt/mesosphere/dcos-csidevices/bin/devices-plugin`

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
The `test_networking.test_vip[Container_POD-Network_HOST-Network_BRIDGE]` failure in `teamcity/dcos/test/docker` looks like a flake: https://jira.mesosphere.com/browse/DCOS-19542

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/docker https://jira.mesosphere.com/browse/DCOS-19542

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by adam-mesos** [comment]
Failures in  teamcity/dcos/test/aws/cloudformation/simple and teamcity/dcos/test/docker/smoke are likely due to infra issues discussed in #tools-infra. I'll re-run them once that if resolved, but we can work on getting reviews now, assuming they'll complete successfully (as they did in previous runs).

**Comment by adam-mesos** [comment]
Actually, the docker/smoke failure in https://teamcity.mesosphere.io/viewLog.html?buildId=901848 looks like https://jira.mesosphere.com/browse/DCOS_OSS-1681 so I'll rerun that now.

**Comment by adam-mesos** [comment]
cloudformation/simple error in https://teamcity.mesosphere.io/viewLog.html?buildId=903754 is definitely the "You have attempted to create more buckets than allowed" issue @mellenburg is investigating in #tools-infra

**Comment by adam-mesos** [comment]
@mesosphere-mergebot override-status 
teamcity/dcos/test/aws/cloudformation/simple QUALITY-1679

**Comment by mellenburg** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/aws/cloudformation/simple QUALITY-1679

**Comment by jgehrcke** [comment]
@mesosphere-mergebot pr-status-checks

**Comment by jgehrcke** [comment]
@mesosphere-mergebot ship_it_label_status_check

**Comment by jgehrcke** [comment]
`mergebot/enterprise/build-status/aggregate` is read although all status checks in the downstream PR are green. `@mesosphere-mergebot pr-status-checks` did not change this situation.

**Comment by jgehrcke** [comment]
@mesosphere-mergebot override-status mergebot/enterprise/build-status/aggregate DCOS-19844

**Pull request merged**